src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.cs.xlf

```diff
@@ -93,8 +93,8 @@
         <note />
       </trans-unit>
       <trans-unit id="AvoidOutParametersTitle">
-        <source>Avoid out parameters</source>
-        <target state="translated">Vyhněte se výstupním parametrům</target>
+        <source>Avoid 'out' parameters</source>
+        <target state="needs-review-translation">Vyhněte se výstupním parametrům</target>
```

Comment:
My change here will most likely prevent the translation of out. I think that's correct, but pointing to this in case that's not what you want.